stac_fastapi/extensions/setup.py

```diff
@@ -1,16 +1,14 @@
 """stac_fastapi: extensions module."""
 
+
 from setuptools import find_namespace_packages, setup
 
 with open("README.md") as f:
     desc = f.read()
 
 install_requires = [
-    "attrs",
-    "pydantic[dotenv]<2",
-    "stac_pydantic==2.0.*",
-    "stac-fastapi.types",
-    "stac-fastapi.api",
+    "stac-fastapi.types==2.4.9",
+    "stac-fastapi.api==2.4.9",
```

Comment:
not really because we will do #648 ;-)

Reply:
The thing is though it's sort of complicated because let's say we put types to 3.0.0 but types has not been release to pypi yet so the tests don't pass.

Reply:
In sfeos I have to push a new core version manually to pypi before I can push es and os updates with new versions

Reply:
we won't change the version before the release

Reply:
this is how we do in titiler and it works fine. 

right now the versioning is quite dangerous because if you have multiple version of the modules you could hit some issues. Because the 3 modules are supper linked to each other, the versions should always be the same

Reply:
the release is triggered by a github workflow - if we put stac-fastapi.types==3.0.0 in stac-fastapi.api then the tests will fail so how can we merge?

Reply:
if you don't change the version before the release then the release will point to old versions

Reply:
I don't think it's dangerous becuase stac-fastapi is run in a container or in the cloud - no one should have old versions installed that won't be re-installed

Reply:
Does anyone use them separately?

Reply:
They all have to be on the same version together at the same time pretty much

Reply:
ok let's discuss this in an issue 👍

Reply:
See what people think